open-media-match/src/OpenMediaMatch/blueprints/development.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+
+"""
+Development only routes for easily testing functionality end-to-end, all running on a single host.
+"""
+
+from flask import Blueprint, abort, redirect, request, url_for
+
+from OpenMediaMatch.blueprints.hashing import hash_media
+from OpenMediaMatch.blueprints.matching import lookup, lookup_signal
+from OpenMediaMatch.utils import abort_to_json, require_request_param
+
+
+bp = Blueprint("development", __name__)
+
+
+@bp.route("/query")
+@abort_to_json
+def query_media():
+    """
+    Hash the input media and then match against all banked content.
+    This is simply a wrapper around the hasing and matching APIs.
+
+     Input:
+     * url - path to the media to hash. Supports image or video.
+     * signal_type - Signal type (hash type)
+
+     Output:
+        * List of matching content items
+    """
+    signal_type_to_signal_map = hash_media()
+    if not isinstance(signal_type_to_signal_map, dict):
+        # We really should not get here, since if something went wrong, it should
+        if isinstance(signal_type_to_signal_map, tuple):
+            return signal_type_to_signal_map
+        abort(500, "Something went wrong while hashing the provided media.")
+
+    # Check if signal_type is an option in the map of hashes
+    signal_type_name = require_request_param("signal_type")
```

Comment:
blocking: I don't think this should be required - should just use all available if unspecified